bin/Debugger.Sample/Debugger.Sample.cpp

```diff
@@ -131,15 +189,34 @@ JsErrorCode RunScript(const wchar_t* filename, JsValueRef* result)
         return JsErrorInvalidArgument;
     }
 
-    // Load the script from the disk.
-    std::wstring script = LoadScript(fullPath);
-    if (script.empty())
+    if (loadScriptUsingBuffer)
     {
-        return JsErrorInvalidArgument;
+        JsValueRef sourceUrl;
+        JsValueRef scriptValueRef;
```

Comment:
(I don't know if my previous comment lost) - initialize this variable with JS_INVALID_REFERENCE